app/src/main/java/com/orange/ouds/app/ui/MainScreen.kt

```diff
@@ -76,22 +80,16 @@ fun MainScreen(themes: List<OudsThemeContract>, mainViewModel: MainViewModel = v
 
 @Composable
 private fun MainScreen(themes: List<OudsThemeContract>, userThemeName: String?, onThemeSelected: (String) -> Unit) {
-    val isSystemInDarkTheme = isSystemInDarkTheme()
     val mainState = rememberMainState(
         themeState = rememberThemeState(
             currentTheme = rememberSaveable { mutableStateOf(getCurrentTheme(userThemeName, themes)) },
-            darkModeEnabled = rememberSaveable { mutableStateOf(isSystemInDarkTheme) },
             availableThemes = themes
         )
     )
 
-    // Change isSystemInDarkTheme() value to make switching theme working with custom color
-    val configuration = LocalConfiguration.current.apply {
-        isDarkModeEnabled = mainState.themeState.darkModeEnabled
-    }
-
+    val isSystemInDarkTheme = isSystemInDarkTheme()
```

Comment:
As we call `isSystemInDarkTheme` now, we can use it in `AboutFileScreen` and rename variable if necessary.